modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.client/src/main/scala/edu/gemini/seqexec/web/client/components/sequence/steps/StepsTableContainer.scala

```diff
@@ -260,7 +260,11 @@ object StepsTableContainer {
           )
         ),
         <.td(
-          if (step.breakpoint) SeqexecStyles.breakpointTrOn else SeqexecStyles.breakpointTrOff,
+          (step.breakpoint, step.skip) match {
+            case (true, true) => SeqexecStyles.breakpointTrOnSkipped
```

Comment:
This would certainly be more readable if we had actual objects rather than booleans